packages/arb-token-bridge-ui/src/util/TokenUtils.ts

```diff
@@ -223,7 +223,7 @@ export async function getL1ERC20Address({
 /*
  Retrieves the L1 gateway of an ERC-20 token using its L1 address.
 */
-export async function getL1GatewayAddress({
+export async function fetchErc20L1GatewayAddress({
```

Comment:
it's calling `erc20Bridger.getL1GatewayAddress` in the function so unless we change the sdk method, I think it's better to align the name with sdk (stick to the old one but maybe add erc20 to the name) `getErc20L1GatewayAddress`?

Reply:
btw in `approveTokenEstimateGas` of `TokenApprovalUtils.ts`, we are calling `erc20Bridger.getL1GatewayAddress` directly

might want to use this method?

Reply:
I'm still kinda for doing `fetch`, we can update the SDK methods in v4?

Reply:
as long as they are the same